index.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const { program } = require("commander");
+
+program
+  .option(
+    "--max-log-lines [number]",
+    "lines of build output to show for each section",
+    50
+  )
+  .parse(process.argv);
+
```

Comment:
```suggestion
```

This part should be in `next-on-netlify.js`, which is the CLI (as indicated by `#!/usr/bin/env node` :slightly_smiling_face:)